### ISSUE TITLE
Doesn't work on ubuntu 14.10 64bit and windows 8.1 64bit

### ISSUE THREAD

**Issue by rets5s** [AtomLinter/Linter#287]
I am running latest version of atom 0.155.0 on Windows and 0.154.0 on Ubuntu. I installed linter and linter-clang, however nothing happens. Is there any prerequisites I am missing?
I tried this code.
```
#include <stdio.h>
asd
int main(){
  return 0
}
```
Dev console didn't show any errors. It showed this.
```
is node executable: false utils.coffee:5
stderr Ignoring /tmp/AtomLinter1141111-3092-1j6vv1usource.c++/File.cpp; not a valid file name (c++)
Done processing /tmp/AtomLinter1141111-3092-1j6vv1usource.c++/File.cpp
Total errors found: 0
```
Am I missing something? I have been trying to figure this stuff out for hours now.

**Comment by hd-deman** [comment]
I think that problem in `++` symbols  in file path.

**Comment by norberteder** [comment]
Running Atom 0.178.0 and Linter 0.11.1 on Ubuntu 14.10 64bit and nothing happens.
